adapter.go

```diff
@@ -148,14 +151,19 @@ func NewAdapter(driverName string, dataSourceName string,params ...interface{})
 	return a, nil
 }
 
-// NewAdapterByDB obtained through an existing Gorm instance get  a adapter, specify the table prefix and the table name
+// NewAdapterByDBUseTableName obtained through an existing Gorm instance get  a adapter, specify the table prefix and the table name
```

Comment:
`obtained through an existing Gorm instance get  a adapter`

This sentense doesn't make sense. Change it to smooth English.